### DISCUSSION TITLE
When we can expect a new release?

### DISCUSSION THREAD

**Discussion by boerge1** [ooflet/Mi-Create#43] [Questions-EN]
The Program is working fine but the last release is outdated. 
Or is there another way to update an existing installation on windows?

**Comment by ooflet** [comment]
I'm finishing up GMF (wfDef.json) support and some new features like AOD editing, animation playback and number previewing in the GMF support branch https://github.com/ooflet/Mi-Create/tree/GMF-format-support-(json), once I'm done with that I will package a new release.

**Reply by ooflet** [reply]
You can update the compiler & data for Mi Create from EasyFace by going into settings, clicking 3 dots menu button on the top bar and selecting the EasyFace folder.

**Reply by boerge1** [reply]
Thanks!
